src/referencestrip.js

```diff
@@ -62,43 +62,107 @@ var THIS = {};
+    // showReferenceStrip:           false,
+    // referenceStripElement:        null,
+    // referenceStripId:             null,
+    // referenceStripScroll:         'horizontal',
+    // referenceStripPosition:       'BOTTOM_LEFT',
+    // referenceStripSizeRatio:      0.2,
+    // referenceStripMaintainSizeRatio: false,
+    // referenceStripTop:            null,
+    // referenceStripLeft:           null,
+    // referenceStripHeight:         null,
+    // referenceStripWidth:          null,
+    // referenceStripAutoResize:     true,
+    // referenceStripAutoHide:       true,
+    // referenceStripAutoHideFactor: 0.5,
+    // referenceStripAutoFade:       true,
+    // referenceStripBackground:     '#000',
+    // referenceStripOpacity:        0.8,
+    // referenceStripBorderColor:    '#555',
+
+    // this.referenceStrip = new $.ReferenceStrip({
+    //     viewer:            this,
+    //     element:           this.referenceStripElement,
+    //     id:                this.referenceStripId,
+    //     scroll:            this.referenceStripScroll,
+    //     position:          this.referenceStripPosition,
+    //     sizeRatio:         this.referenceStripSizeRatio,
+    //     maintainSizeRatio: this.referenceStripMaintainSizeRatio,
+    //     top:               this.referenceStripTop,
+    //     left:              this.referenceStripLeft,
+    //     width:             this.referenceStripWidth,
+    //     height:            this.referenceStripHeight,
+    //     autoResize:        this.referenceStripAutoResize,
+    //     autoHide:          this.referenceStripAutoHide,
+    //     autoHideFactor:    this.referenceStripAutoHideFactor,
+    //     autoFade:          this.referenceStripAutoFade,
+    //     background:        this.referenceStripBackground,
+    //     opacity:           this.referenceStripOpacity,
+    //     borderColor:       this.referenceStripBorderColor,
+    //     //TODO
+    //     //crossOriginPolicy: this.crossOriginPolicy,
+
+    //     viewer:      this,
+    //     id:          this.referenceStripId,
+    //     scroll:      this.referenceStripScroll,
+    //     position:    this.referenceStripPosition,
+    //     sizeRatio:   this.referenceStripSizeRatio,
+    //     height:      this.referenceStripHeight,
+    //     width:       this.referenceStripWidth
+    // });
+
+    const viewer      = options.viewer,
+          viewerSize  = $.getElementSize( viewer.element );
 
     //We may need to create a new element and id if they did not
     //provide the id for the existing element
     if ( !options.id ) {
-        options.id              = 'referencestrip-' + $.now();
-        this.element            = $.makeNeutralElement( "div" );
-        this.element.id         = options.id;
-        this.element.className  = 'referencestrip';
+        options.id                   = 'referencestrip-' + $.now();
+        this.stripElement            = $.makeNeutralElement( "div" );
```

Comment:
I have mixed feelings about this rename. It's nice to be more specific, but it goes against our tradition (e.g. the viewer has an `element` property). Also, `element` is an option that can get passed in, which, again, is the same in other places. Since we're inside the reference strip, adding `strip` to our element seems redundant. If nothing else, I'm not seeing where the `element` gets put into the `stripElement` property. 

What are your thoughts on it?

Reply:
Hi @iangilman !

My thoughts are...this is way too early of a review for this and there's kind of a method to my madness lol

The only reason I changed the name was I copy/pasted code over from navigator to make the positioning options, and corresponding implementation, consistent between the two. Before pasting, I wanted any existing .element references clearly different than the .element references that got pasted. The final name will be element ;)

Reply:
I love it. Always good to have a method to your madness! Carry on, and thank you for the explanation 😊